Tests/FluentAssertions.Equivalency.Specs/SelectionRulesSpecs.cs

```diff
@@ -1493,86 +1493,131 @@ public void Including_an_interface_property_through_inheritance_should_work()
 
         [Fact]
         public void Including_non_browsable_members_when_testing_NotEquivalent_should_work_when_browsable_field_differs()
```

Comment:
Almost ready...

We need to avoid to use the literal name of APIs like `NotBeEquivalentTo` and `BeEquivalentTo`. In this case, I would focus the scenarios using `BeEquivalentTo` because that's what `NotBeEquivalentTo` internally uses.  E.g. `Browsable_fields_can_be_included_in_the_assertion`

Also, I would explicitly show the value of `BrowsableField` of the subject.

Also applies on the other scenarios.

Reply:
What do you mean by

> explicitly show the value of `BrowsableField` of the subject

?? The test should write it to test output??

Reply:
[Fact]
        public void When_browsable_field_differs_including_non_browsable_members_should_not_affect_result()
        {
            // Arrange
            var subject = new ClassWithNonBrowsableMembers() { BrowsableField = 2 };
            var expectation = new ClassWithNonBrowsableMembers() { BrowsableField = 1 };

            // Act
            Action action =
                () => subject.Should().BeEquivalentTo(expectation, config => config.IncludingNonBrowsableMembers());

            // Assert
            action.Should().Throw<XunitException>();
        }

Reply:
Ahh, okay, don't use the implicit value of 0. Got it :-)

Reply:
I've pushed a commit to this effect. It explicitly assigns to the member where a difference is benig set up for both the subject and the expectation now.

Reply:
> Ahh, okay, don't use the implicit value of 0. Got it :-)

Exactly, because you don't know that if you look at the test. In other words, if it's important to understand the test case, it's very important to show it ;-)